contracts/subdaos/cwd-subdao-timelock-single/src/contract.rs

```diff
@@ -176,17 +176,25 @@ pub fn execute_execute_proposal(
             .querier
             .query_wasm_smart(proposal_module, &ProposalQueryMsg::Config {})?;
 
+        // expect exactly one `ExecuteMsg::ExecuteTimelockedMsgs` message
+        if proposal.msgs.len() != 1 {
```

Comment:
Let's explicitly add a check somewhere here that the message is really `ExecuteMsg::ExecuteTimelockedMsgs` and nothing else